lib/ansible/cli/galaxy.py

```diff
@@ -402,8 +402,13 @@ def execute_install(self):
             # query the galaxy API for the role data
 
             if role.install_info is not None and not force:
-                display.display('- %s is already installed, skipping.' % role.name)
-                continue
+                if role.install_info['version'] != role.version:
```

Comment:
insufficient check, this can downgrade a role, which is not what most will expect